css/custom.scss

```diff
@@ -330,7 +330,10 @@ $link-active: $link-hover;
 .peoples {
   margin-top: 1em;
 }
-
+.code {
```

Comment:
Wave still complains about the contrast error: 
![image](https://user-images.githubusercontent.com/1842467/198210085-836ffa70-c5cc-43e0-94bc-e5b80f4740b2.png)

Reply:
Hi!
I just made a commit and reverted the changes.